packages/js/data/src/countries/resolvers.ts

```diff
@@ -58,6 +60,24 @@ export function* getCountries() {
 	}
 }
 
+export function* getCurrencySymbols() {
+	try {
+		const url = 'wc/v3/data/currencies';
+		const results = yield apiFetch( {
+			path: url,
+			method: 'GET',
+		} );
+
+		return getCurrencySymbolsSuccess(
+			results.map(
+				( currency ) => `${ currency.code }: ${ currency.symbol }`
```

Comment:
I set the store currency to NTD (New Taiwan Dollar), but the currency symbol is not displayed correctly. I believe this is because we store the currency symbols as an array, not an object.

I think we should convert it to match the same structure as the `wcSettings.admin.onboarding.currencySymbols`.

Reply:
https://github.com/user-attachments/assets/915c1855-97cf-40af-a6b0-270e84b59bd4

Reply:
Good catch! Fixed it in [Correct currencySymbol format](https://github.com/woocommerce/woocommerce/pull/55464/commits/fabfeb631e654185c585d3f6ae794ac839353700)